client/my-sites/customer-home/cards/tasks/task.jsx

```diff
@@ -158,11 +163,16 @@ const Task = ( {
 					) }
 				</div>
 			</div>
-			{ isDesktop() && (
+			{ isDesktop() && illustration && (
 				<div className="task__illustration">
 					<img src={ illustration } alt="" />
 				</div>
 			) }
+			{ isDesktop() && remainingTasks && totalTasks && (
```

Comment:
Rather than adding card-specific elements to the generic `Task` component, could we instead pass in a full component for the visual (so, for instance, the `Gauge` import would be in the card itself)? It would be nice to keep the `Task` component as applicable to all cards as possible.

Reply:
Thank you for your review @kwight. I applied the changes you mentioned in the commit 7856135